<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipointsDemarcate</class>
 <widget class="QDialog" name="MultipointsDemarcate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>710</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>多点标定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxInstallMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>190</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelInstallMode">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相机安装方式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_CameraInstallMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>430</width>
     <height>220</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBoxCoordinate">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>190</width>
       <height>200</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="btnMoveRight">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>右</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveDown">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>下</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveLeft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>左</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveUp">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>上</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelCurPointCoordinate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>(100,100)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelCurPointCoordTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>VGA坐标:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxPointNum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="btnPointNumDec">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPointNumInc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPointNum">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNext">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPrev">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCurPointIndex">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelCurPointTitle">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>点选择</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelPointNumTitle">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>标定点数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_PointInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QComboBox" name="comboBox_PointStep">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>170</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPointStep">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>170</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>点任务</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelPointResult">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>90</y>
        <width>170</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>点结果</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_PointResult">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>170</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_LineInfo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QComboBox" name="comboBox_LineStep">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>170</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelLineStep">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>170</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>线任务</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LineResult">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>170</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelLineResult">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>90</y>
        <width>170</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>线结果</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_Data">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>351</width>
     <height>701</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="labelTitilePixTable">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>像素坐标</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetPixel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>330</width>
       <height>280</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="labelTitlePhyTable">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>物理坐标</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetPhysical">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>330</width>
       <height>280</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnImportTestData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>120</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
     <property name="text">
      <string>导入测试数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartDemarcate">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>650</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
     <property name="text">
      <string>开始标定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeleteLastOne">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>650</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
     <property name="text">
      <string>删除末项</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearAllData">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>650</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
     <property name="text">
      <string>全部清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="btnGetOnePointAxisR">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
     <property name="text">
      <string>抓取</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeleteLastOneAxisR">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
     <property name="text">
      <string>删除末项</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearAllDataAxisR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
     <property name="text">
      <string>全部清空</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_AxisR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>330</width>
       <height>161</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>140</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <row>
      <property name="text">
       <string>点1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>点2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>点3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>结果</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>--</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>--</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>--</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>--</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>--</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>--</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>--</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>--</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_ArmPicDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>330</width>
       <height>100</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="labelArmPicDir">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>机械手与图像方向</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnArmPicDirSame">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>相同</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnArmPicDirReverse">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>相反</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCalc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>650</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCenterAxisR">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>5</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>R轴中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDemarcateData">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>5</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>标定数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
